.github/workflows/release-code-freeze.yml

```diff
@@ -77,53 +77,80 @@ jobs:
+      #             }
+      #           },
+      #           {
+      #             "type": "section",
+      #             "text": {
+      #               "type": "mrkdwn",
+      #               "text": "If you absolutely need to include a PR that was not merged in `develop` yet, please consult the release lead."
+      #             }
+      #           },
+      #           {
+      #             "type": "section",
+      #             "fields": [
+      #               {
+      #                 "type": "mrkdwn",
+      #                 "text": "*Version:*\n${{ env.RELEASE_VERSION }}"
+      #               },
+      #               {
+      #                 "type": "mrkdwn",
+      #                 "text": "*Planned date:*\n<https://wcpay.wordpress.com/dev-resources/wc-payments-release-calendar/|${{ env.RELEASE_DATE }}>"
+      #               }
+      #             ]
+      #           }
+      #         ]
+      #       }
+
+      - name: Add comment to PR
+        uses: actions/github-script@v6
         with:
-          # For posting a rich message using Block Kit (https://api.slack.com/messaging/interactivity)
-          payload: |       
-            {
-                "blocks": [
-                    {
-                        "type": "header",
-                        "text": {
-                            "type": "plain_text",
-                            "text": "Code freeze notification",
-                            "emoji": true
-                        }
-                    },
-                    {
-                        "type": "section",
-                        "text": {
-                          "type": "mrkdwn",
-                          "text": "The automated workflow just created a release branch and raised a Pull Request to merge changes on trunk. Any PRs that were not already merged will be a part of the release after that by default.\n If you have something that needs to be released in ${{ env.RELEASE_VERSION }}, please consult the release lead."
-                        }
-                    },
-                    {
-                        "type": "section",
-                        "fields": [
-                            {
-                                "type": "mrkdwn",
-                                "text": "*Version:*\n<${{ env.GITHUB_TAG_URL }}|${{ env.RELEASE_VERSION }}>"
-                            },
-                            {
-                                "type": "mrkdwn",
-                                "text": "*Planned date:*\n<https://wcpay.wordpress.com/dev-resources/wc-payments-release-calendar/|${{ env.RELEASE_DATE }}>"
-                            }
-                        ]
-                    }
-                ]
-            }
+          script: |
+            github.rest.issues.createComment({
```

Comment:
Using [gh pr comment](https://cli.github.com/manual/gh_pr_comment) directly could be easier and avoid introducing a 3rd party action

Not sure if you're just experimenting on that PR or if you plan also to ship this, just wanted to mention again that I didn't expect this in this PR, so it's fine if you decide to revert this part

Reply:
I was just experimenting stuff :) to see if it works.

Reply:
`gh pr` command works fine. I've removed the action.